### ISSUE TITLE
Not connecting to chat-gpt when running chatgpt_airsim.py

### ISSUE THREAD

**Issue by Snorlax104** [microsoft/PromptCraft-Robotics#8]
I have been attempting to try the drone simulation but when I run the chatgpt_airsim.py it gives error. I tried reinitializing everything and performed all basic debugging still no success. 

![image](https://user-images.githubusercontent.com/97780276/224672718-11f4a6b5-b9f4-465e-b8d9-9ba3a35ae900.png)

**Comment by gullipalli-suneetha** [comment]
I am also getting the same error. Someone help me to fix it. Thank you

**Comment by saihv** [comment]
Hi, is the AirSim simulation running along side? Can you see the environment and the drone initialized properly?

**Comment by gullipalli-suneetha** [comment]
Hello, Good morning,

No it's not working. When i gave .\run.bat this command (not sure about
this command if it is wrong i am sorry) opening AirSim simulator with
drone. But after running chatgpt_airsim.py file getting errors.

Thanking you,
Suneetha Gullipalli.

**Comment by Snorlax104** [comment]
> Hi, is the AirSim simulation running along side? Can you see the environment and the drone initialized properly?

Yes the simulation is running in the background and I can see the drone and environment working.

**Comment by gullipalli-suneetha** [comment]
@Snorlax104 Chatgpt_aursim.py file running successfully without any errors?

**Comment by Snorlax104** [comment]
> @Snorlax104 Chatgpt_aursim.py file running successfully without any errors?

No the above screenshot is for the same. These are the errors I am getting when I run chatgpt_airsim.py.

**Comment by uiseoklee** [comment]
I also had the same problem as above. After putting the access_token in the config.json file, I ran airsim and ran chatgpt_airsim.py. I also take a screenshot of my problem.

![image](https://user-images.githubusercontent.com/51503303/225244231-fee91cfd-c383-4350-94e2-2429cbf5e25f.png)

**Comment by shamitb** [comment]
Upgrade the revChatGPT package: pip install revChatGPT - it worked fine after this for me!

**Comment by gullipalli-suneetha** [comment]
after upgrade also getting same error.

**Comment by huazai665** [comment]
I am also getting the same error.

**Comment by yuan-alex** [comment]
Connection was failing due to a change in the revChatGPT package's proxy. The latest release now uses the official OpenAI API.

**Comment by huazai665** [comment]
> 

How to fix the program? pip install revChatGPT of old version?

**Comment by huazai665** [comment]
@yuan-alex thank you. I can run the demo successfully. It is amazing.

**Comment by langxinspieder** [comment]
Initializing ChatGPT...
Done.
Initializing AirSim...
no question response！ what should I do？

**Comment by WangLouis** [comment]
Hi~ 
I have been attempting to try the drone simulation but when I run the chatgpt_airsim.py it gives error. I tried reinitializing everything and performed all basic debugging still no success.
Does anyone have suggestions? thanks~~ ![Image](https://github.com/user-attachments/assets/55d051d2-9183-49e1-9e24-e3a97dea039f)

**Comment by Xj1567589354** [comment]
no problem

**Comment by Xj1567589354** [comment]
no problem
